File: python/Cats and a Mouse #101.py
import math
def catAndMouse(A,B,C):
    if abs(A-C)>abs(B-C):
        print("Cat B")
    elif abs(A-C)<abs(B-C):
        print("Cat A")
    else:
        print("Mouse C")

N = int(input())
for _ in range(N):
    X = list(map(int,input().split()))
    catAndMouse(X[0],X[1],X[2])

File: Ruby/Cats_and_a_Mouse_#101.rb
def catAndMouse(x, y, z)
  if (x - z).abs < (y - z).abs
    return "Cat A"
  elsif (x - z).abs > (y - z).abs
    return "Cat B"
  else
    return "Mouse C"
  end
end

q = gets.to_i
for i in 1..q do
    xyz = gets.rstrip.split
    x = xyz[0].to_i
    y = xyz[1].to_i
    z = xyz[2].to_i
    result = catAndMouse x, y, z
    puts result
end
